pybind11/how-to.rst

```diff
@@ -220,6 +220,12 @@ We wrap the class using the ``py::class_<T>`` template:
 
     As in the example, classes should almost always have a :ref:`shared_ptr holder type <style-guide-pybind11-holder-type>`.
 
+.. note::
+
+    It is far more common to name the pybind11 class ``cls`` and wrap one class per module or per function within a module.
+    However, the examples in this document are quite distributed, so to make them easier to follow we use ``clsExampleOne``.
+    See also :ref:`class prefix <style-guide-pybind11-class-prefix>`.
```

Comment:
Hmm... I'm not sure that this clarifies things. The style guide doesn't say anything about "one class per function": it seems to imply that wrapping more than one class per file means you *have* to use `clsExample`.

Reply:
I looked at the style guide entry and I think `wrapper` in this case can refer to either a function or the full wrapper, as appropriate. The text should be cleaned up to clarify that. I suppose I could do that on this ticket, though I wonder if that needs an RFC.

Reply:
I also do one class per function, as it lets you group similar classes in one file, but keep them distinct.

Probably doesn't need an RFC and do it on this ticket. Give it to Pim or Jim or Krzysztof to review, since they're the ones who would care the most. A brief review of our existing code suggests its what we're doing already in many places.